src/emmy/structure.cljc

```diff
@@ -291,6 +296,41 @@
                 (.-v s))
          (print-method w))))
 
+#?(:cljs
+   (defn make-es6-indexable
```

Comment:
This would be a good candidate for `^:no-doc` metadata so it doesn't appear in the public cljdoc api docs

Reply:
✅